instance/models.py

```diff
@@ -147,6 +156,7 @@ class WriteitInstancePopitInstanceRecord(models.Model):
         )
     writeitinstance = models.ForeignKey(WriteItInstance)
     popitapiinstance = models.ForeignKey(ApiInstance)
+    popolo_source = models.ForeignKey(PopoloSource, null=True, blank=True)
```

Comment:
I'm getting an error on this line when I run the tests:

```
(writeit-virtualenv)vagrant@precise64:/vagrant$ ./manage.py test nuntium contactos mailit
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute
    django.setup()
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/__init__.py", line 21, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/apps/config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/vagrant/instance/models.py", line 149, in <module>
    class WriteitInstancePopitInstanceRecord(models.Model):
  File "/vagrant/instance/models.py", line 159, in WriteitInstancePopitInstanceRecord
    popolo_source = models.ForeignKey(PopoloSource, null=True, blank=True)
NameError: name 'PopoloSource' is not defined
```

I'm guessing this line has snuck into this pull request unintentionally? :)